src/engine/blocks.js

```diff
@@ -567,6 +567,16 @@ class Blocks {
                 if (!optRuntime){
                     break;
                 }
+                // The selected item in the sensing of block menu needs to change based on the
+                // selected target.  Set it to the first item in the menu list.
```

Comment:
Do you mind filing a tech debt issue and then referencing it here via `// TODO (#<issue>)`

Reply:
(The one that we talked about in the original PR)